src/draw/sw/lv_draw_sw.c

```diff
@@ -75,19 +78,22 @@ void lv_draw_sw_init(void)
     lv_draw_sw_mask_init();
 #endif
 
-    uint32_t i;
-    for(i = 0; i < LV_DRAW_SW_DRAW_UNIT_CNT; i++) {
-        lv_draw_sw_unit_t * draw_sw_unit = lv_draw_create_unit(sizeof(lv_draw_sw_unit_t));
-        draw_sw_unit->base_unit.dispatch_cb = dispatch;
-        draw_sw_unit->base_unit.evaluate_cb = evaluate;
-        draw_sw_unit->base_unit.delete_cb = LV_USE_OS ? lv_draw_sw_delete : NULL;
-        draw_sw_unit->base_unit.name = "SW";
+    lv_draw_sw_unit_t * draw_sw_unit = lv_draw_create_unit(sizeof(lv_draw_sw_unit_t));
```

Comment:
I believe this can be a improvement on the SW side as the draw units were basically identical: e.g. the same dispatcher and the same evaluate.

On our case, in a RT1170 for example, we have both PXP and VGLite draw units but those have different implementation for dispatcher and evaluation callbacks.
Of course, we can merge them in a single set of callbacks, but the code will look messy - at the end there are anyway two draw threads. Not sure about the gain we got by having a single evaluation for each task, should not have so much effect.